core/templates/css/oppia.css

```diff
@@ -4516,3 +4524,211 @@ div.oppia-issues-learner-action {
 .oppia-rule-preview {
   word-wrap: break-word;
 }
+
+.oppia-welcome-guide-block {
```

Comment:
Ditto here and elsewhere (Reason: I don't think these style are general/common style and we goona use it in other pages]

Reply:
fixed